docs/datamodel/indexes.rst

```diff
@@ -51,7 +51,7 @@ Similarly indexes may refer to the link properties if the *subject* is a link:
 
     abstract link friends_base {
         property nickname -> str;
-        index nickname_idx on (__subject__@nickname);
+        index on (__subject__@nickname);
```

Comment:
I keep forgetting — do we really need `(` / `)` around the `expr` part?

Reply:
https://github.com/edgedb/edgedb/blob/f9a73382b0e08634cfec423ef608c630a835d521/edb/edgeql/parser/grammar/commondl.py#L115-L118